config/core/configmaps/autoscaler.yaml

```diff
@@ -130,3 +130,6 @@ data:
     # Scale to zero grace period is the time an inactive revision is left
     # running before it is scaled to zero (min: 30s).
     scale-to-zero-grace-period: "30s"
+
+    # Enable graceful scaledown feature flag
```

Comment:
Please expand a bit here, what does this mean.

Reply:
done.